ibis/pandas/decimal.py

```diff
@@ -52,7 +52,7 @@ def execute_decimal_log2(op, data, **kwargs):
         return decimal.Decimal('NaN')
 
 
-@execute_node.register(ops.UnaryOp, decimal.Decimal)
+@execute_node.register((ops.UnaryOp, ops.Negate), decimal.Decimal)
```

Comment:
[Isn't](https://github.com/ibis-project/ibis/blob/master/ibis/expr/operations.py#L310) `Negate` a `UnaryOp`?

Reply:
I mean We should prefer the dict lookups, just could mention it in a comment or at the top of the modules.